azure-pipelines.yml

```diff
@@ -25,12 +22,68 @@ resources:
     name: SkillsFundingAgency/das-platform-building-blocks
     ref: refs/heads/master
```

Comment:
We should be using a release tag for this

Reply:
I have updated this to point to the tag for the now merged building block branch